src-built-in/components/myApps/src/components/AppDefinition.jsx

```diff
@@ -57,7 +60,7 @@ export default class AppDefinition extends React.Component {
 
 	render() {
 		const app = this.props.app;
-		if (typeof (app.icon) === "undefined") app.icon = DEFAULT_APP_ICON;
+		if (typeof (app.icon) === "undefined" || app.icon === null) app.icon = DEFAULT_APP_ICON;
```

Comment:
```suggestion
		if (!app.icon) app.icon = DEFAULT_APP_ICON;
```
Why wouldn't that work? It would cover cases where icon is empty string as well.

Reply:
not sure -- maybe false is valid icon? Not sure - don't really have the insight from whoever wrote this initially. Leaving because I don't want to break anything. :-)